synthesizer/src/process/execute_fee.rs

```diff
@@ -56,30 +56,29 @@ impl<N: Network> Process<N> {
         #[cfg(feature = "aleo-cli")]
         println!("{}", format!(" • Calling '{}/{}'...", request.program_id(), request.function_name()).dimmed());
 
-        // Initialize the execution.
-        let execution = Arc::new(RwLock::new(Execution::new()));
-        // Initialize the inclusion.
-        let inclusion = Arc::new(RwLock::new(Inclusion::new()));
+        // Initialize the trace.
+        let trace = Arc::new(RwLock::new(Trace::new()));
```

Comment:
```suggestion
        let trace = Default::default();
```
(type inference should be able to handle this, same with `metrics`)